### ISSUE TITLE
Daily PPA broken for Ubuntu 16.10

### ISSUE THREAD

**Issue by nolsen42** [mumble-voip/mumble-ubuntu-ppa#5]
The Daily PPA for Ubuntu 16.10 is missing dependencies, looking for old dependencies that existed on older systems but was upgraded?

```
nolsen@nolsen-HP-Notebook:~$ sudo apt install mumble
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 mumble : Depends: libprotobuf9v5 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Comment by acz13** [comment]
Hmm, libprotobuf9v5 is only a package in 16.04

**Comment by nolsen42** [comment]
This has been fixed now.

**Comment by acz13** [comment]
Yay finally i can use mumble.

**Comment by mkrautz** [comment]
Not yet fixed for 1.2.x / stable PPA.

**Comment by Kissaki** [comment]
16.10 is EOL, so closing this. See other tickets for more up to date PPA stuff. I'll hopefully be able to take a look soon.
